src/image.tsx

```diff
@@ -80,9 +82,9 @@ const Image = React.forwardRef<HTMLImageElement, ImageProps | DeprecatedImagePro
             width = props.width;
             height = props.height;
         } else if (props.width !== undefined) {
-            height = props.width / RATIO[aspectRatio];
+            height = typeof props.width === 'number' ? props.width / ratio : props.width;
```

Comment:
when the height/width is a string it could be better to set it using calc, so the aspect ratio is preserved:
```js
typeof props.width === 'number' ? props.width / ratio : `calc(${props.width} / ratio)`
```

then, instead of passing the width/height as attributes to the image, it should be set using `style` or with jss